### ISSUE TITLE
Set the Session Scope Costom Dimmention

### ISSUE THREAD

**Issue by vwxyutarooo** [MatteoGabriele/vue-analytics#20]
As this title, I'd like no set Session Scope Custom Dimention. I's not working When I trying as following

router/index.js
```
Vue.use(VueAnalytics, { id: 'UA-xxxxxxxx-xx', router });
Vue.$ga.set(['dimension1', 'pc']);
...
export default router;
```

I think it should has set before sending initial pageview and `vue-analtics` does not supported yet.
Is it correct?

**Comment by MatteoGabriele** [comment]
yeah true. now it's not possible.
Thinking on making available a function that is fired before the first hit so it's possible to add everything needed.

**Comment by MatteoGabriele** [comment]
I just moved the `onAnalyticsReady` callback. now it fires before the first hit so you can do stuff there.

published v3.0.2
